sdk/eventhub/Azure.Messaging.EventHubs/samples/Program.cs

```diff
@@ -137,6 +381,34 @@ private static void DisplayHelp()
             Console.WriteLine();
         }
 
+        /// <summary>
+        ///   Reads the selection of the application's user from the
+        ///   console for samples.
+        /// </summary>
+        ///
+        /// <param name="samples">The available samples.</param>
+        ///
+        /// <returns>The validated selection that was made.</returns>
+        ///
+        private static int? ReadSelection(IReadOnlyList<IEventHubsSample> samples)
+        {
+            return ReadSelection(samples.Count() + extraOptionsForSamples.Length);
+        }
+
+        /// <summary>
+        ///   Reads the selection of the application's user from the
+        ///   console for identity samples.
+        /// </summary>
+        ///
+        /// <param name="samples">The available identity samples.</param>
+        ///
+        /// <returns>The validated selection that was made.</returns>
+        ///
+        private static int? ReadSelection(IReadOnlyList<IEventHubsIdentitySample> samples)
```

Comment:
You could make these into expression bodied methods, but I leave that to your discretion.   I don't have a strong opinion.